Underline Strikethrough.roboFontExt/lib/Underline StrikeThrough.py

```diff
@@ -310,7 +310,19 @@ def updateFontList(self):
                             dictionary.update({fontIdentifier: value})
                         else:
                             dictionary.update({fontIdentifier: None})
-                        
+                # Deal with underline position
+                if 'public.openTypePostUnderlinePosition' in font.lib.keys():
+                    self.underlinePosition.update{fontIdentifier: font.lib['public.openTypePostUnderlinePosition'}
+                # If font doesn't have that lib key set, but is using fontmake, use fontmake behavior
+                elif font.lib["com.robofont.fontCompilerTool"] == 'fontmake':
```

Comment:
Do these two do the same thing?
```
from mojo.UI import getDefault
print(getDefault("fontCompilerTool"))

f = CurrentFont()
print(f.lib["com.robofont.fontCompilerTool"])
```
Got a `KeyError` for the 2nd.

Also:
Can we make the metadata work for both compilers? What if someone has `afdko` set as their compiler, but is preparing the font metadata for later generation with `fontmake` through Terminal?

Reply:
I'm trying to think of other ways to intuit the intending meaning of the position. I guess this is the best way.

Reply:
Well, once ufo2ft is updated, it'll use the public key for `post` and you'll get the correct value